modules/prod/kogito-jobs-service-postgresql/added/kogito-app-launch.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
can't we reuse upstream modules in these cases? The files look identical.

Reply:
@ricardozanini It would be possible to use the `kogito-app-launch.sh` from upstream images for all prod images. I guess I can just change `ADDED_DIR="${SCRIPT_DIR}"/added` in `configure` to use the path to the file used in upstream, wdyt?

My only concern is if some images move to different repository related to Apache upstream release, but if you think that it will not be a problem, I can change that.

Reply:
As agreed with Ricardo via meeting, we will keep like this for now. 

In the future we can improve this part, probably by splitting some modules in community and some of those modules can be used in the product (e.g the module containing the `kogito-app-launch.sh` scripts).